forward_engineering/helpers/alterScriptHelpers/columnHelpers/alterComputedColumnHelpr.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+module.exports = (app, ddlProvider) => {
+	const _ = app.require('lodash');
```

Comment:
Please use the lodash dep from the [plugin modules](https://github.com/hackolade/SQLServer/blob/349ec7a277851d3bb3b0c13d7943ef23f8ff19bf/package.json#L64).